### PR TITLE
 Mention proper way to install without using the binary cache

### PR DESCRIPTION
**Pull request by jonasnick**

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -122,13 +129,14 @@ This is borrowed from the [NixOS manual](https://nixos.org/nixos/manual/index.ht
...
-	wget https://releases.nixos.org/nixos/18.09/nixos-18.09.2257.235487585ed/nixos-graphical-18.09.2257.235487585ed-x86_64-linux.iso
+	wget https://releases.nixos.org/nixos/19.03/nixos-19.03.172361.cf3e277dd0b/nixos-minimal-19.03.172361.cf3e277dd0b-x86_64-linux.iso
 	```
+	Alternatively you can build NixOS from source by following the instructions at https://nixos.org/nixos/manual/index.html#sec-building-cd.
 
 2. Write NixOS iso to install media (USB/CD). For example:
 
 	```
-	dd if=nixos-graphical-18.09.2257.235487585ed-x86_64-linux.iso of=/dev/sdX
+	dd ifnixos-minimal-19.03.172361.cf3e277dd0b-x86_64-linux.iso
```
Comment (nixbitcoin) [inline docs/install.md]: `=` missing and why did you remove `of=/dev/sdX` ?

Comment (jonasnick) [reply docs/install.md]: fixed

**Comment by jonasnick** [comment]
Rebased. Did your test succeed @nixbitcoin ?

**Comment by nixbitcoin** [comment]
Yes, but only with a custom fix to neo.map. Waiting for https://github.com/NixOS/nixpkgs/pull/60201 to merge. If this PR is stuck I will submit my own fix.

**Comment by nixbitcoin** [comment]
Looks like all changes have been backported to 19.03! Will test building without substitutes again soon.

thin-provisioning-tools: https://github.com/NixOS/nixpkgs/commit/2e6afa8cb127662efa77df27fdbacce6d3b7aca4
pythonPackages.rednose: https://github.com/NixOS/nixpkgs/commit/c21f08bfedde0f7d00a5ebc85455fb28b9037932
neo.map: https://github.com/NixOS/nixpkgs/commit/3366e982e8143e2b8fa5e02818ed86d4ec3ba918
pinentry: https://github.com/NixOS/nixpkgs/commit/918b4e294fed68d1ef50900714208616385f2c8a
